app/actions/remote/entry/app.ts

```diff
@@ -72,6 +72,13 @@ export async function appEntry(serverUrl: string, since = 0) {
     if (rolesData.roles?.length) {
         modelPromises.push(operator.handleRole({roles: rolesData.roles, prepareRecordsOnly: true}));
     }
+
+    // clear lastUnreadChannelId
+    const removeLastUnreadChannelId = prepareCommonSystemValues(operator, {lastUnreadChannelId: ''});
+    if (removeLastUnreadChannelId) {
+        modelPromises.push(removeLastUnreadChannelId);
+    }
```

Comment:
1. I thought we decided to remove the record instead of adding an empty string.
2. This should happen for every server and for every type of entry, so I think it would be best to do this on `init/launch` for every server.
3. this should also happen on team switch

Reply:
That is not enough and that is server change not team change.

Again, clearing/deleting should happen on entry for every server, best place to do that is in app launch, as that takes care for all entry cases

Reply:
For example, when the app is opened from a push notification -00Entry is kot used, if you tap on an in-app notification from another server is also not used, switching to a team is also not clearing the unread value, and probably the same will be tru for permalink, deeplinks and so on...